lib/brains/client.py

```diff
@@ -10,14 +10,17 @@
 
 log = getLogger('s.brains')
 
-# Note: between the double slash is where the BRAINTREE_MERCHANT_ID goes. In
-# tests this will be blank, hence the double slash.
-mocks = {
-    'POST /merchants//client_token': (200, get_sample('token')),
-    'GET /merchants//customers/does-not-exist': (404, ''),
-    'GET /merchants//customers/minimal': (200, get_sample('customer-minimal')),
-    'GET /merchants//customers/full': (200, get_sample('customer-full')),
-}
+mocks = {}
```

Comment:
maybe a comment above this to explain that a test is responsible for populating it